### PR TITLE
Added Aaron's comment about underscored report names during update.

### LINKED ISSUE #1335: Site update preserves modified reports
If you modify a standard report, then during update, the site renames it with an underscore prefix (`_`). This behavior makes sure that it's not overwritten by the standard report during the update process.

(Internal reference [BUG 3896252](https://msazure.visualstudio.com/Configmgr/_workitems/edit/3896252))

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ce5953cb-562f-3eab-9fcc-8d7b987ac025
* Version Independent ID: 2b5ddaf1-b68c-b8cc-cf65-9327779cce64
* Content: [Reporting best practices - Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/servers/manage/best-practices-for-reporting)
* Content Source: [sccm/core/servers/manage/best-practices-for-reporting.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/servers/manage/best-practices-for-reporting.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### PR DESCRIPTION
**Pull request by ChrisKibble**
Fixes #1335

### CODE REVIEW

**Comment by PRMerger14** [comment]
@ChrisKibble : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/core/servers/manage/best-practices-for-reporting.md [hunk 1]
```diff
@@ -19,13 +19,16 @@ ms.collection: M365-identity-device-management
...
 
 ## Optimize SQL Server Reporting Services queries  
- Typically, any reporting delays are because of the time it takes to run queries and retrieve the results. If you are using Microsoft SQL Server, tools such as Query Analyzer and Profiler can help you optimize queries.  
+Typically, any reporting delays are because of the time it takes to run queries and retrieve the results. If you are using Microsoft SQL Server, tools such as Query Analyzer and Profiler can help you optimize queries.  
 
 ## Schedule report subscription processing to run outside standard office hours  
- Whenever possible, schedule report subscription processing to run outside normal office standard hours to minimize the CPU processing on the Configuration Manager site database server. This practice also improves availability for unpredicted report requests.  
+Whenever possible, schedule report subscription processing to run outside normal office standard hours to minimize the CPU processing on the Configuration Manager site database server. This practice also improves availability for unpredicted report requests.  
+
+## Updating Built-In Reports
```
[the remark this answers is not in the document]
Comment (ChrisKibble) [reply sccm/core/servers/manage/best-practices-for-reporting.md]: @aczechowski - sorry, I'm missing the change between the two blocks in this one...

[the remark this answers is not in the document]
Comment (aczechowski) [reply sccm/core/servers/manage/best-practices-for-reporting.md]: Not in the block, just the H2 on line 30. Switch to `## Updating built-in reports` 😄

**Comment by mestew** [comment]
#sign-off Thanks @ChrisKibble

**Comment by ktoliver** [comment]
Publishing is suspended until Monday, November 4. Merging will resume 7 AM Pacific Monday. 

This PR is ready for merge.

**Comment by ktoliver** [comment]
Turns out this can be merged now.

**Pull request merged**